libs/gretty/src/main/groovy/org/akhikhl/gretty/GrettyPlugin.groovy

```diff
@@ -89,6 +89,7 @@ class GrettyPlugin implements Plugin<Project> {
 
     project.dependencies {
       grettyStarter "org.gretty:gretty-starter:$grettyVersion"
+      grettyStarter ProjectUtils.getCurrentGroovy(project).dependencies
```

Comment:
@f4lco - I'm using [gradle-versions-plugin](https://github.com/ben-manes/gradle-versions-plugin/) to check for dependency updates. Ever since updating to Gretty 3.0.7 which contains this change, I started getting these "possible updates":

```
- org.codehaus.groovy:groovy-cli-commons [3.0.8 -> 3.0.9]
    https://groovy-lang.org
- org.codehaus.groovy:groovy-json [3.0.8 -> 3.0.9]
    https://groovy-lang.org
```

I don't have these libraries in my project. I believe this is caused by the line I'm commenting on. Why was it added, are you sure that the rest of the changes in the PR are not enough for what it tries to fix?